src/main/java/com/aws/greengrass/mqttclient/AwsIotMqttClient.java

```diff
@@ -81,6 +81,11 @@
     // Limit bandwidth to 512 KBPS
     private final RateLimiter bandwidthLimiter = RateLimiter.create(512.0 * 1024);
 
+    // Limit TPS to 1 which is IoT Core's limit for connect requests per client-id
+    // Connect throttled at 1 TPS setting the limit to .25
```

Comment:
explain why `0.25` instead of 1.

Reply:
Added a explanation and a TODO for follow up